docs/using-availability-zones.md

```diff
@@ -0,0 +1,220 @@
+# Availability Zones
+
+**Feature Status:** Alpha since v1.12.
```

Comment:
Is there a feature gate that users have to set to enable this?

Reply:
Noop, this is enabled for v1.12, so that new zoned nodes could be joined easily after upgrading an existing cluster.